css-basics_social-card/css-basics_button-styles/css/styles.css

```diff
@@ -0,0 +1,67 @@
+/* Please scroll down to the start of your exercise! */
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  height: 100vh;
+}
+
+section {
+  background-color: lemonchiffon;
+  display: grid;
+  place-items: center;
+}
+
+.skyblue {
+  background-color: skyblue;
+}
+
+button {
+  background-color: white;
+  padding: 8px 25px;
+}
+
+/* Your exercise starts here */
+
+/* Add your CSS styles below this line. Feel free to replace the comments with your code. Make sure not to put your code into the comments, because then nothing will happen. */
+
+.button1 {
+  border-radius: 2rem;
+  background-color: gold;
+  color: goldenrod;
+  font-size: large;
+  /*Style me! Change my border, set a border radius or what you want to try out. :) */
+}
+
+.button2 {
+  background-color: #116644;
+  color: azure;
+  padding: 1rem;
+  border-radius: 50%;
+  font-size: large;
+  /*Style me! Change my color, or my background-color. Or both! You can test out different color modes here - like HEX, RGB, ... */
+}
+
+.button3 {
+  font-family: cursive;
```

Comment:
Did this work? Font-family usually is the font (e.g. Times New Roman), not the style/decoration of the text. But not sure, you might check that again.